File: sla/sla.r
op <- readline("Digite uma operacao (+ - * /): ")
a <- as.numeric(readline("Digite o primeiro numero: "))
b <- as.numeric(readline("Digite o segundo numero: "))

resultado <- switch(op,
  "+" = a + b,
  "-" = a - b,
  "*" = a * b,
  "/" = if (b != 0) a / b else "Erro: divisao por zero!",
  "Operacao invalida!"
)

cat("Resultado:", resultado, "\n")

File: sla/sla.m
op = input('Digite uma operacao (+ - * /): ', 's');
a = input('Digite o primeiro numero: ');
b = input('Digite o segundo numero: ');

switch op
    case '+'
        resultado = a + b;
    case '-'
        resultado = a - b;
    case '*'
        resultado = a * b;
    case '/'
        if b ~= 0
            resultado = a / b;
        else
            resultado = 'Erro: divisao por zero!';
        end
    otherwise
        resultado = 'Operacao invalida!';
end

disp(['Resultado: ', num2str(resultado)])
